types/enum.json

```diff
@@ -134,14 +134,14 @@
     "weapon": {
       "type": "string",
       "enum": [
-        "ARTILLERY_CANON",
-        "BAZOOKA",
-        "LIGHT_TANK_CANON",
-        "MACHINE_GUN",
-        "MEDIUM_TANK_CANON",
-        "ROCKETS",
-        "TANK_MACHINE_GUN",
-        "VUCLAN_CANNON"
+      "MACHINE_GUN",
```

Comment:
the identation changed here, If we start having big json files, maybe we should try to add some json formatter? to keep the identation and everything standard? (only an idea)

Reply:
Yeah i think so too